### ISSUE TITLE
It seems the VR device "Pico 4" is not supported.

### ISSUE THREAD

**Issue by YLyeliang** [Okabintaro/SubmersedVR#87]
As the title described, any solutions to resolve this?

**Comment by YoDudeNoWay** [comment]
Here's what I did to make it work on mine:

- I unplugged my Series X controller from the PC
- Deleted all \common\Subnautica files, including BepIndex and SubmersedVR
- Reinstalled BepIndex through Vortex instead of dropping the files in the main Subnautica folder like I had done previously
- Reinstalled SubVR by dropping the files in the main folder like you usually do
- Don't forget -vrmode and openvr on the launch options

And that's the fix I encountered. I don't really know if it was fresh install that did the trick or just unplugging the controller from the computer, since i'd launch Subnautica and it'd load the standard VR port, which used the controller. I still could join a save by using Pico 4's controllers just fine, but the other buttons such as the joysticks wouldn't work. Maybe it was a conflict that was caused by the controller being plugged in? Anyway, it works just fine now. Hope that helps!

**Comment by NeonPixels** [comment]
I use the Pico 4 through SteamVR, and it worked perfectly on the first try.

This might mean there's some configuration issue on your end. Sorry to not be of much help, but I wanted to mention that it has worked correctly in my case.

**Comment by Anna3863** [comment]
> Here's what I did to make it work on mine:
> 
> * I unplugged my Series X controller from the PC
> * Deleted all \common\Subnautica files, including BepInEx and SubmersedVR
> * Reinstalled BepInEx through Vortex instead of dropping the files in the main Subnautica folder like I had done previously
> * Reinstalled SubVR by dropping the files in the main folder like you usually do
> * Don't forget -vrmode and openvr on the launch options
> 
> And that's the fix I encountered. I don't really know if it was fresh install that did the trick or just unplugging the controller from the computer, since i'd launch Subnautica and it'd load the standard VR port, which uses the controller. I still could join a save by using Pico 4's controllers just fine, but the other buttons such as the joysticks wouldn't work. Maybe it was a conflict that was caused by the controller being plugged in? Anyway, it works just fine now. Hope that helps!

i did all of this and it still doesnt work not even chatgpt knows what to do anymore

**Comment by Raccamoon** [comment]
TLDR: Try to use "-vrmode openvr" launch options with Virtual Desktop if it does not work properly with Pico headsets.

This issue is old, but still open so gonna post the fix here. Depending on how you have connected your Pico it might not work immediatly. Pico is a bit iffy with OpenXR so e.g. Virtual Desktop still uses OperVR (older standard) for Pico headsets. Checking the BepInEx log the mod said that the headset was not supported and to use "-vrmode openvr" launch code. This fixed the issue for me at least. If this does not resolve the issue you can try other streaming software (ALVR, Pico Connect, SteamLink) since they might handle OpenXR/VR differently
